packages/patrol_cli/lib/src/compatibility_checker.dart

```diff
@@ -0,0 +1,119 @@
+import 'dart:async';
+
+import 'package:dispose_scope/dispose_scope.dart';
+import 'package:file/file.dart';
+import 'package:patrol_cli/src/base/constants.dart';
```

Comment:
import this as `constants`, so we can do `constants.version`. It's easier to see where the `version` comes from this way.